worksheet_C/README.md

```diff
@@ -0,0 +1,19 @@
+a) This algorithm checks if there is a duplicate in the list. It does this by starting with the first item in the list and checking, in order, every other item in the list to see if they match. It then does this with the second item. When it finds a duplicate it returns true. 
+
+B)  Because there are is a for loop within a for loop (a nested loop). This means in the worst case the algorithm will have to check through n*n times to find the duplicate.  
+
+C) It is still correct because every value of j is still checked with every value of I.  
+
+D) The algorithm will run twice as fast because it will not check the same list indexes twice like the original algorithm would. For example  in the original algorithm, if I = 5 and j = 4 it would check if list[5] = list[4] which is the same as if I = 4 and j= 5.  
+
+E) No its not quadratic because it no longer has to check through n*n times.  
```

Comment:
However it has to check n*(n-1)/2 times, and O(n*(n-1)/2) = O((n^2 - n)/2) = O(n^2) because n^2 dominates n and we can ignore the factor of 1/2. So the algorithm is still quadratic, despite being twice as fast.